book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rental
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @label = []
+  end
+
+  def add_rental(rentals)
+    @rentals.push(rentals)
```

Comment:
- [ ] Kindly pass the `person` and `date` to the the `add_rental` method, you can use the following:
```
  def add_rental(date, person)
    @rentals.push(Rental.new(date, self, person))
  ...
```